src/DataBuilder.php

```diff
@@ -818,27 +830,35 @@ protected function getRequestBody()
         return $this->requestBody;
     }
 
-    /*
+    /**
+     * Get the user's IP, by inspecting the http header X-Real-IP, or if not
+     * that first address from the http header X-Forwarded-For, and if not that
+     * then the remote IP connecting to the web server, if available.
+     *
```

Comment:
Other SDKs (e.g. Pyrollbar, Gem) handle it in the opposite way with X-Forwarded-For checking first.

Reply:
@bxsx Would you like this to switched? If so, I recommend a separate task and prioritization.